tests/caasp/stack_bootstrap.pm

```diff
@@ -105,10 +106,14 @@ sub bootstrap {
     assert_and_click "velum-bootstrap";
 
     # Wait until bootstrap finishes
-    assert_screen [qw(velum-bootstrap-done velum-api-disconnected)], 900;
-    if (match_has_tag('velum-api-disconnected')) {
-        setup_root_ca;
+    if (is_caasp '3.0+') {
         assert_screen 'velum-bootstrap-done', 900;
+        setup_root_ca;
+    }
+    else {
+        assert_screen 'velum-api-disconnected', 300;
+        setup_root_ca;
+        assert_screen 'velum-bootstrap-done', 600;
```

Comment:
Taking a look on verification runs I have a feeling that setup_root_ca can be done in same sequence in both scenarios and can be done unconditionally, and do velum-api-disconnected assertion unless (is_caasp '3.0+'). If it's not the case we still can extract common setup_root_ca call, and do another conditional assertion for caasp version lower than 3.0, but that doesn't bring much value =)

Reply:
Certificates are generated on CaaSP 2 after velum_api_disconnected message shows.
There is no such message on CaaSP 3 so I import after bootstrap.